src/components/header/Header.js

```diff
@@ -20,21 +20,21 @@ function Header() {
           <li>
             <a href="#skills">Skills</a>
           </li>
+          <li>
+            <a href="#opensource">Open Source</a>
+          </li>
           <li>
             <a href="#projects">Projects</a>
           </li>
           <li>
-            <a href="#opensource">Open Source</a>
+            <a href="#achievements">Achievements</a>
           </li>
           <li>
             <a href="#blogs">Blogs</a>
           </li>
           <li>
             <a href="#talks">Talks</a>
           </li>
-          <li>
-            <a href="#achievements">Achievements</a>
-          </li>
```

Comment:
Can you remove all changes in this file? This bug is supposed to provide work to new people. If we did all the work, we wouldn't be able to build a community that way. Ask before you start on an issue.

Reply:
OK done.